modules/local/virusbreakend/main.nf

```diff
@@ -6,8 +6,8 @@ process VIRUSBREAKEND {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/gridss:2.13.2--h50ea8bc_3':
-        'quay.io/biocontainers/gridss:2.13.2--h50ea8bc_3' }"
+        'https://pub-29f2e5b2b7384811bdbbcba44f8b5083.r2.dev/singularity/gridss:2.13.2--1' :
```

Comment:
where is this?

Reply:
This is hosted on Cloudflare R2 in the same account as the oncoanalyser reference data, which is currently administered by our group. I'm not aware of an nf-core equivalent to quay.io for Singularity sif files (I think we should look into this!) and I'm aiming here to preserve the ability for users to run oncoanalyser fully with Singularity.

Reply:
I see, in such a case, since Singularity can convert docker container, we usually just use the docker declaration and forget about the other.
But happy to raise the question up

Reply:
Got it, much less of a need for nf-core having a Singularity repo in that case.

So the preference would then be to remove the self-hosted Singularity sif file and let that be handled via Docker image conversion?

Reply:
yeah, let's do this as this is what we do usually in such case.
cf https://github.com/nf-core/modules/blob/master/modules/nf-core/deepvariant/main.nf

Reply:
Sounds good!